src/java/com/zimbra/qa/selenium/projects/admin/pages/PageZextraAdmin.java

```diff
@@ -0,0 +1,237 @@
+		public static final String DelegatedEditbtn = "css=div[id='ztab__ZxAdmin']:not([aria-hidden='true']) table[id='ztabv__ZxAdmin_delegatedAdmins_table'] td.ZWidgetTitle:contains('Edit')";
+		public static final String DelegatedDeletebtn = "css=div[id='ztab__ZxAdmin']:not([aria-hidden='true']) table[id='ztabv__ZxAdmin_delegatedAdmins_table'] td.ZWidgetTitle:contains('Delete')";
+		public static final String Network_NG_ICON="css=div.ImgZeXtras";
+		public static final String DeleteDialogHeader = "css=div[class='DwtDialog']:not([aria-hidden='true']) td[class='DwtDialogTitle']:contains('Delete permissions')";
+		public static final String DeleteDialogYesbtn = "css=div[class='DwtDialog']:not([aria-hidden='true']) td[class='ZWidgetTitle']:contains('Yes')";
+		public static final String zSaveDailog = "css=div.DwtDialog[style*='display: block;'] table tr:contains('Saving Settings')";
+		public static final String zSavedialogClosebtn = "css=div.DwtDialog[style*='display: block;'] td.ZWidgetTitle:contains('Close')";
+		public static final String zResetDomainDialog = "css=div[class='DwtDialog']:not([aria-hidden='true']) td[class='DwtDialogTitle']:contains('Reset Domain settings')";
+		public static final String zResetDomainDialogYesbtn = "css=div[class='DwtDialog']:not([aria-hidden='true']) 	td[class='ZWidgetTitle']:contains('Yes')";
+		public static final String zDomainResetbtn = "css=div[id='ztab__ZxAdmin']:not([aria-hidden='true']) table[id='ztabv__ZxAdmin_domainSettings_table'] td.ZWidgetTitle:contains('Reset')";
+		public static final String zDomainEditbtn = "css=div[id='ztab__ZxAdmin']:not([aria-hidden='true']) table[id='ztabv__ZxAdmin_domainSettings_table'] td.ZWidgetTitle:contains('Edit')";
+		public static final String zCompleteResetDialogOkbtn = "css=div[class='DwtDialog']:not([aria-hidden='true']) td[class='ZWidgetTitle']:contains('OK')";
+	}
+
+	public PageZextraAdmin(AbsApplication application) {
+		super(application);
+		logger.info("new " + myPageName());
+	}
+
+	@Override
+	public void zNavigateTo() throws HarnessException {
+
+		if (zIsActive()) {
+			return;
+		}
+
+		SleepUtil.sleepLong();
+		sClickAt(Locators.Network_NG_ICON, "");
+		zWaitForWorkInProgressDialogInVisible();
+		sClickAt(Locators.Admin, "");
+		zWaitForWorkInProgressDialogInVisible();
+
+	}
+
+	@Override
+	public boolean zIsActive() throws HarnessException {
+
+		// Make sure the Admin Console is loaded in the browser
+		if (!MyApplication.zIsLoaded())
+			throw new HarnessException("Admin Console application is not active!");
+
+		boolean present = sIsElementPresent(Locators.AdminTab);
+		if (!present) {
+			return (false);
+		}
+
+		boolean visible = zIsVisiblePerPosition(Locators.DelegatedAddbtn, 0, 0);
+		if (!visible) {
+			logger.debug("isActive() visible = " + visible);
+			return (false);
+		}
+
+		return (true);
+	}
+
+	public AbsWizard zPressButton_Account(Button button) throws HarnessException {
```

Comment:
Let's have zPressButton instead of zPressButton_Account, zPressButton_Domain and pass account, domain etc.. as second argument. We can do any other way around as well, please rethink better way.

Reply:
I would suggest like this,
1. Keep zPressButton button instead of adding 2nd argument to use simple way in future.
2. Create separate buttons like B_EDIT_ACCOUNT, B_EDIT_COS, B_REMOVE_COS etc..
3. In zPressButton method put condition based on button to use exact locator.